webapp/src/ts/services/db-sync.service.ts

```diff
@@ -391,22 +395,24 @@ class DbSyncTelemetry {
   private readonly failedToParse = 'Unexpected token';
   private readonly database;
   private readonly direction;
-  private readonly start;
+  private readonly endLastReplicated;
   private readonly lastReplicated;
   private readonly key;
-  private end;
+  private trackReplication;
 
   constructor(
-    public telemetryService:TelemetryService,
+    public telemetryService: TelemetryService,
+    public performanceService: PerformanceService,
     database,
     direction,
     lastReplicated?,
   ) {
     this.database = database;
     this.direction = direction;
-    this.start = Date.now();
     this.lastReplicated = lastReplicated;
+    this.endLastReplicated = Date.now(); // The lastReplicated is based on Date API, so we use the Date API here too.
```

Comment:
The `lastReplicated` is used in the About page too, I decided not to change the Date API to make the calculation using the same thing (apples with apples)